presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorSplitManager.java

```diff
@@ -15,10 +15,21 @@
 
 import com.facebook.presto.spi.ConnectorSession;
 import com.facebook.presto.spi.ConnectorSplitSource;
+import com.facebook.presto.spi.ConnectorTableHandle;
 import com.facebook.presto.spi.ConnectorTableLayoutHandle;
 
 public interface ConnectorSplitManager
 {
+    default ConnectorSplitSource getSplits(
+            ConnectorTableHandle tableHandle,
```

Comment:
The parameter order of the old `getSplits` makes more sense to me.

Reply:
The idea is to use tableHandle represent connector's view of table (physical positions, enforced predicates(what today's Layout handles), ... etc.) and use trait/property to represent all these constraints (tupledomain, ...), but we are not there yet.